tests/nuodb_test_class.py

```diff
@@ -34,15 +35,36 @@ def assert_await(fxn, timeout=120, interval=10):
 
     if last_assert:
         raise last_assert
+    
+def getClient():
+     influxToken = os.getenv("INFLUXDB_TOKEN", 'none')
+     org = os.getenv("INFLUXDB_ORG", "nuodb")
+     client = InfluxDBClient(url='http://localhost:8086', token=influxToken, org=org)
+     return client
+
+def getMeasurenment(client, bucket):
+    query = f"""
+    import \"influxdata/influxdb/schema\"
```

Comment:
Are the quotes here and and below escaped for InfluxDB or for Python? When I test this, it looks like the escape character is actually gratuitous, since the multi-line string is delimited by `"""` and `"` is just a normal character:

```
>>> print("""
... "test"
... """)

"test"

>>> print("""
... \"test\"
... """)

"test"
```